draft-packages/empty-state/KaizenDraft/EmptyState/README.mdx

```diff
@@ -133,10 +135,10 @@ Your CTA button should compliment the title of the empty state. If a user scans
 Based on the overall user flow and the nature of the empty state, make sure you adjust your tone accordingly. A positive empty state should contain encouraging language, while an action empty state should get to the point quickly. Here are some helpful tips for each type of empty state:
 
 *   **Positive empty state** - Include words of affirmation here. If a user has nothing left to do after this point, let them know that. Be careful though, as this type of language can sometimes come off as condescending.
-*   **Informative empty state** - Reassure the user that work is being done in the background and that we’ll notify them when there’s data on that page for them to see or action. This is an opportunity to educate.
+*   **Informative empty state** - Reassure the user that work is being done in the background and that we’ll notify them when there’s data on that page for them to see or action. This is not used as a holding state to replace data that is in the process of being updated. This is an opportunity to inform.
```

Comment:
This is a little confusing to me. What do we mean by "This is not used as a holding state..."? It contradicts the first sentence when we tell the user we'll notify them when there's something for them to do - isn't that kind of a holding state? Is letting the user know that there's data being updated not informing them?

Reply:
Ah the idea we're trying to convey is that is shouldn't be used to replace existing data while it's being updated or something. How about if I remove the reference to holding state like "This is not to be used to replace existing data that is in the process of being updated."?

Reply:
That change helps, @dlimiter